.github/ISSUE_TEMPLATE/create-new-doctype-template.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Create new doctype template
+about: For issues where a templateer needs to create a new doctype template.
+title: Create {doctype} template
+labels: templates
+assignees: ''
+
+---
+
+# Create {doctype} template
+
+## Description
+### Doctype Name: {doctype}
+### Directory: https://github.com/thegooddocsproject/templates/{directory_name}
+### Sub-tasks
+1. Create {doctype}-template.md and write content for it
+2. Create {doctype}-guide.md and write content for it
+
+{Optional: add any further explanation if the issue is not self-explanatory}
+
+## Helpful References
+{Optional: Add links to anything that might help the templateer write this particular type of template. For example, a Medium article about how to write good tutorials.}
+
+## Before You Submit
+Add an appropriate label to denote the relative priority of this template:
+
+Label | Description
+--|--
```

Comment:
**Question:** I'm not sure what this section does inside GitHub. Does it automatically assign these labels to the issue?

Reply:
No, it's just renders a little table, telling the creator of the issue which of our many labels they should assign specifically for priority. They would choose one and assign it using the normal label process. If you weren't familiar with our scale, you wouldn't know whether we ranked from 0-4 or 4-0, and what criteria you should use for choosing the priority.